src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -124,13 +124,25 @@ export class DiscussionThread {
     }, {});
   }
 
+  private streamIDs = new Set();
+
   private async updateCommentsThreadUponMessageArrival(comment: Types.Message): Promise<void> {
```

Comment:
Nit: 
```suggestion
  private async updateCommentsThreadUponMessageArrival(commentStreamMessage: Types.Message): Promise<void> {
```

I was confused, and expected the type `IComment` based on naming of param `comment`